src/client.rs

```diff
@@ -10,7 +10,76 @@ use std::collections::HashSet;
 //     GameOver,
 // }
 
+pub struct Scoreboard {
+    // timer: u32,
+    found_mines: u32,
```

Comment:
a bit about design of this struct: u mix logic (found mines, total mines) with grafics (position, message),

otherwise, we already counting mines in field. so, imo, logic may be separated to another struct, and Scoreboard should be updated and drawn with params from this struct.

another thing: we alrdy have client serv-struct, and later we wanna have multiplayer, not sure about how yet, though.. (again, now to just see that logic works its fine probably)